Products/Poi/events.py

```diff
@@ -86,7 +85,11 @@ def remember_tracker(new_issue, event):
 
 
 def assign_id(new_issue, event):
-    """Auto-increment ID numbers"""
+    """Auto-increment ID numbers when they are created.
+       Don't run on copied issues
+    """
+    if new_issue.id.find('copy') >= 0:
+        return
```

Comment:
I'm sure it could be disabled, I just didn't like the idea of disabling something that used to work. As it is now, copying an issue behaves the same as it used to.